pageserver/src/pgdatadir_mapping.rs

```diff
@@ -1551,6 +1552,8 @@ impl<'a> DatadirModification<'a> {
     pub async fn commit(&mut self, ctx: &RequestContext) -> anyhow::Result<()> {
         let mut writer = self.tline.writer().await;
 
+        let timer = WAL_INGEST.time_spent_on_ingest.start_timer();
```

Comment:
Technically this is a hot code path, but it's probably fine. Can you create a follow up to remove this when you're done with it?

Reply:
yep I plan to remove it two weeks later

Reply:
Let's open an issue for it. I always forget to come back to this sort of stuff :smile: